components/program/Timeline.vue

```diff
@@ -0,0 +1,355 @@
+<script setup lang="ts">
+  import type { AlertType } from 'composables/useAlerts'
+
+  const useAuth = useAuthStore()
+
+  const companies = await $fetch('/api/company', { method: 'GET' })
```

Comment:
If you want SSR for this component you have to use the `useFetch` composable: https://nuxt.com/docs/getting-started/data-fetching